src/resources/js/admin-upgrade-page.js

```diff
@@ -0,0 +1,39 @@
+tribe.upgradePage =  tribe.upgradePage || {};
+
+( function ( $, obj ) {
+	'use strict';
+
+	obj.setup = function() {
+		if ( $( '#current-settings-tab' ).val() != 'upgrade' ) {
+			// if it is already enabled, we don't need to show the button
+			if ( tribe_upgrade.v2_is_enabled == "1" ) {
+				return;
+			}
+
+			$( '.tribe_settings > h1' ).append( '<button id="upgrade-button">✨ Upgrade your calendar views</button>' );
+			$( document ).on( 'click', '#upgrade-button', function( e ) {
```

Comment:
Can't we make that button into a link? And avoid this action?

Reply:
It'd be a nice improvement, but the change isn't required in order to merge.